client/src/util/fetchFromAPI.js

```diff
@@ -13,17 +13,14 @@ export const queryAPI = async (options) => {
     throw new Error("onError is not provided");
   }
   const result = {
-    data: null,
-    error: null,
+    data: "",
+    error: "",
```

Comment:
Tell me more about this change?

Reply:
I forgot to revert it back to `null` instead of that empty string.